docs/Coding.md

```diff
@@ -0,0 +1,206 @@
+#Coding Standards
+
+The Strimzi UI projects uses best practices based off the official [React TypeScript Cheat sheet](https://react-typescript-cheatsheet.netlify.app/), with modifications for this project. The React TypeScript Cheat sheet is maintained and used by developers through out the world, and is a place where developers can bring together lessons learned using TypeScript and React.
+
+## Imports
+
+Since we are using TypeScript 4.x + for this project, default imports should conform to the new standard set forth in TypeScript 2.7:
+
+```javascript
```

Comment:
javascript -> typescript ?

Reply:
This comment applies for all snippets here - could they all be updated?

Reply:
yes should be typescript.